internal/cmd/run/run.go

```diff
@@ -127,6 +127,7 @@ func Command() *cobra.Command {
 	sc.StringToString("experiment", "sauce::experiment", map[string]string{}, "Specifies a list of experimental flags and values")
 	sc.Bool("dry-run", "dryRun", false, "Simulate a test run without actually running any tests.")
 	sc.Int("retries", "sauce::retries", 0, "Retries specifies the number of times to retry a failed suite (sauce mode only)")
+	sc.String("launch-by", "sauce::launchBy", "", "Launch jobs by test history data, e.g. fail_rate.")
```

Comment:
Better explanation 🙏 Also, this flag itself is not necessarily tied to test history data. The actual parameter is. And even then, it's Job history, not test history.